templates/home/index.html.twig

```diff
@@ -5,22 +5,58 @@
 {% block stylesheets %}
     <link href="https://fonts.googleapis.com/css?family=Lato:100" rel="stylesheet" type="text/css">
     {{ parent() }}
-    <style>
-        body {
-            margin: 0;
-            padding: 0;
-            font-weight: 100;
-            font-family: 'Lato';
-        }
 
-        .title {
-            margin: 20px auto;
-            text-align: center;
-            font-size: 96px;
-        }
-    </style>
 {% endblock %}
 
 {% block body %}
-    <div class="title">NFQ Akademija</div>
+   <ul class="list-group m-4">
+       <li class="list-group-item list-group-item-info">Studentai</li>
+       {% for key, project in projects %}
+           {% for student in project['students'] %}
+               <li class="list-group-item">
+                   <a href="{{ path('students') }}?name={{ student }}&project={{key }}">{{ student }}</a>
```

Comment:
`path` komanda gali priimti argumentus. Nes adrese rašyti `&` nėra gera praktika.
Pavyzdyje buvo `&amp;` šioje vietoje.

Plačiau:
* https://symfony.com/doc/current/templating.html#linking-to-pages (žr. pavyzdė su `'slug': article.slug`)
* https://twig.symfony.com/doc/2.x/filters/url_encode.html